data_utils/pr_comments.py

```diff
@@ -127,46 +141,68 @@ def process_comment(comment):
         if not comment["author"]:
             comment["author"] = {"login": "deleted-user"}
 
+        is_host = comment["author"]["login"] in host_usernames
+
         return [
             comment["resourcePath"], comment["createdAt"],
-            comment["author"]["login"], comment["body"]
+            comment["author"]["login"], comment["body"],
+            repo_type, is_host
         ]
     return None
 
 
 def main():
     """ Retrieves the comments in pull requests for intern repositories.
 
-    The comments retrieved depend on command line arguments. Repositories
-    of different types are stored in different CSV files. Current supported
-    repository types are "starter" and "capstone".
-
-    <repo_type>_pr_comments.csv has the following columns:
+    pr_comments.csv has the following columns:
         comment_path: The resource path to the comment.
         created: The date and time that the comment was created.
         author: The Github username of the comment author.
         comment: The text in the comment.
+        repo_type: The type of repository that the comment was made in.
+        is_host: Boolean that indicates if author is a host.
     """
-
-    try:
-        repo_type = sys.argv[1]
-    except:
-        raise Exception("Usage: pr_comments.py <repository type>")
-
-    repo_csv = f"data/{repo_type}_repos.csv"
+    arg_count = len(sys.argv)
+
+    # Check if in testing mode
+    if arg_count == 1:
```

Comment:
at some point having this be a bit more ... clear might be helpful, ahah

Reply:
I changed the comment a bit. Let me know if you think it's still a bit unclear!